src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarArtists.tsx

```diff
@@ -94,13 +94,17 @@ export class ArtworkSidebarArtists extends React.Component<ArtistsProps> {
 export const ArtworkSidebarArtistsFragmentContainer = createFragmentContainer(
   ArtworkSidebarArtists,
   graphql`
-    fragment ArtworkSidebarArtists_artwork on Artwork {
+    fragment ArtworkSidebarArtists_artwork on Artwork
+      @argumentDefinitions(
+        showFollowSuggestions: { type: "Boolean", defaultValue: true }
+      ) {
       artists {
         __id
         id
         name
         href
         ...FollowArtistButton_artist
+          @arguments(showFollowSuggestions: $showFollowSuggestions)
```

Comment:
Would have been even nicer if we could just do `@arguments(showFollowSuggestions: true)`, but [currently](https://github.com/facebook/relay/pull/2501) the compiler insists on only using variables.